app/controllers/games_controller.rb

```diff
@@ -33,6 +33,13 @@ def games_not_found
     @slug = slug_param
   end
 
+  def leave_game
+    @game = Game.find_by(slug: slug_param) or games_not_found
+    @game.players.delete(session_player)
+    if @game.players.empty? then @game.destroy end
```

Comment:
Another opportunity for a callback on the model.

Reply:
This might be too much complication at this stage, but if instead of destroying the game you were to mark the game with a status 'trashed', that would allow us to give a message 'That game has ended' on website.com/game/123/show, as opposed to 'That game doesn't exist'. Only if you feel like it.

Reply:
In the lobby-page-2 branch I have added a migration for 'status', so once that is merged, we can do `game.status = 'trashed'` (and update set_game to account for this - shouldn't be returning any trashed games).